examples/tests/crawl_and_generate_SBOM.py

```diff
@@ -0,0 +1,84 @@
+import os
```

Comment:
This is a nit. [PEP 8](https://peps.python.org/pep-0008/#imports) in Python states this about imports:

> Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

I *think* the correct ordering of imports would be this:

```suggestion
import json
import os
import re
import subprocess
import tempfile
from urllib.parse import urlparse

import requests
from git import Repo
```